src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const list = document.querySelector('.gallery__list');
+const largeImg = document.querySelector('.gallery__large-img');
+
+list.addEventListener('click', function(e) {
```

Comment:
Краще явно вказувати, адже це може бути і `error`
```suggestion
list.addEventListener('click', function(event) {
```